PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -294,6 +294,27 @@ local function CreateCamera()
 
 	function this:Update()
 	end
+	
+	function this:ApplyVRPosition()
+		--we only want this to happen in first person VR
+		local player = game.Players.LocalPlayer
+		if not player or not player.Character 
+				or not player.Character:FindFirstChild("HumanoidRootPart") 
+				or not player.Character.HumanoidRootPart:FindFirstChild("RootJoint") then
+			return
+		end
+		
+		if not UserInputService.VREnabled or not this:IsInFirstPerson() then
```

Comment:
Would it be better if I moved the check for VR out of that function? I'm not sure how long this is going to stick around, but I agree that it should be as low-impact to non-VR as possible.